test/end-to-end/reports/debtor_account_balance/balanceReport.spec.js

```diff
@@ -0,0 +1,46 @@
+/* global browser, element, by */
+
+const chai = require('chai');
+const helpers = require('../../shared/helpers');
+
+helpers.configure(chai);
+
+const DebtorClientAccountBalanceReportPage = require('./balance.page');
+
+describe('Debtor Clients Account Balance Report', () => {
+  let Page;
+  const key = 'debtorBalanceReport';
+
+  const dataset = {
+    fiscal_id : 2,
```

Comment:
The goals of an end-to-end test is to try and mimic the behavior of a user.  A user will not select a fiscal year by the id - they will select it by text.  Try putting the text here and updating your `bhFiscalSelect` component to handle that case.